sdk-web/src/main/java/com/reach5/identity/sdk/customTab/ReachFiveLoginActivity.kt

```diff
@@ -0,0 +1,91 @@
+package com.reach5.identity.sdk.customTab
+
+import android.app.Activity
+import android.content.ComponentName
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.util.Log
+import androidx.browser.customtabs.*
+import com.reach5.identity.sdk.core.utils.Pkce
+import com.reach5.identity.sdk.customTab.ConfiguredCustomTabProvider.Companion.PKCE
+
+
+class ReachFiveLoginActivity : Activity() {
+    private val CUSTOM_TAB_PACKAGE_NAME = "com.reach5.identity.sdk.customtab"
+    private val TAG = "Reach5"
+
+    private var authCode: String? = null
+
+    private var mCustomTabsServiceConnection: CustomTabsServiceConnection? = null
```

Comment:
Could we rename these? :)

Reply:
Are you ok with these changes?

- `mCustomTabsServiceConnection` -> `customTabsConnection`
- `mClient` -> `customTabsClient`
- `mCustomTabsSession` -> `customTabsSession`

Reply:
pas de prefix `m`